src/lib/y2network/presenters/interfaces_summary.rb

```diff
@@ -50,6 +53,115 @@ def text
+      #
+      # @example with many configured interfaces
+      #   presenter.one_line_text
+      #   => "Multiple Interfaces"
+      #
+      # @return [String] summary in just one line and in plain text
+      def one_line_text
+        protocols = []
+        output = []
+
+        protocols << "DHCP" if config.connections.any?(&:dhcp?)
+        protocols << "STATIC" if config.connections.any?(&:static?)
+
+        output << protocols.first if protocols.uniq.size == 1
+
+        case config.connections.size
+        when 0
+          return Yast::Summary.NotConfigured
+        when 1
+          output << config.connections.first.interface
+        else
+          output << _("Multiple Interfaces")
+        end
+
+        output.join(" / ")
+      end
+
+      # Generates a summary in RichText format for the configured interfaces
+      #
+      # @example
+      #   interfaces_summary.proposal_text
+      #   => "<ul><li><p>Configured with DHCP: eth0, eth1<br></p></li>" \
+      #      "<li><p>br0 (Bridge)<br>IP address: 192.168.122.60/24" \
+      #      "<br>Bridge Ports: eth2 eth3</p></li></ul>"
+      #
+      # @see Summary
+      # @return [String] summary in RichText
+      def proposal_text
+        items = []
+        items << list_item_for(dhcp_summary) unless dhcp_ifaces.empty?
+        items << list_item_for(static_summary) unless static_ifaces.empty?
+        items << list_item_for(bridge_summary) unless bridge_connections.empty?
+        items << list_item_for(bonding_summary) unless bonding_connections.empty?
+        items << list_item_for(Yast::Summary.NotConfigured) if items.empty?
+
+        Yast::Summary.DevicesList(items)
+      end
+
+      # Return a summary of the interfaces configurew with DHCP
+      #
+      # @return [String] interfaces configured with DHCP summary
+      def dhcp_summary
+        # TRANSLATORS: %s is the list of interfaces configured by DHCP
+        _("Configured with DHCP: %s") % dhcp_ifaces.sort.join(", ")
+      end
+
+      def dhcp_ifaces
+        config.connections.select(&:dhcp?).map(&:interface)
+      end
+
+      # Return a summary of the interfaces configured statically
+      #
+      # @return [String] statically configured interfaces summary
+      def static_summary
+        # TRANSLATORS: %s is the list of interfaces configured by DHCP
+        _("Statically configured: %s") % static_ifaces.sort.join(", ")
+      end
+
+      # Return a summary of the configured bridge interfaces
+      #
+      # @return [String] bridge configured interfaces summary
+      def bridge_summary
+        _("Bridges: %s") % bridge_connections.map do |connection|
+          "#{connection.name} (#{connection.ports.sort.join(", ")})"
+        end
+      end
+
+      # Return a summary of the configured bonding interfaces
+      #
+      # @return [String] bonding configured interfaces summary
+      def bonding_summary
+        _("Bonds: %s") % bonding_connections.map do |connection|
```

Comment:
As above, I miss the translators comment

Reply:
Same than above.